sdk/storage/azblob/zz_generated_connection.go

```diff
@@ -51,7 +51,15 @@ func newConnection(endpoint string, cred azcore.Credential, options *connectionO
 	policies = append(policies, options.PerCallPolicies...)
 	policies = append(policies, runtime.NewRetryPolicy(&options.Retry))
 	policies = append(policies, options.PerRetryPolicies...)
-	policies = append(policies, cred.NewAuthenticationPolicy(runtime.AuthenticationOptions{TokenRequest: policy.TokenRequestOptions{}}))
+
+	switch c := cred.(type) {
```

Comment:
Seems like a `default` clause it appropriate here, although that would mean this func would need to return the resultant error.